src/IB/IBFEInstrumentPanel.cpp

```diff
@@ -581,7 +581,7 @@ IBFEInstrumentPanel::initializeHierarchyDependentData(IBFEMethod* ib_method_ops,
         // update FE system data for meter_mesh
         initializeSystemDependentData(ib_method_ops, jj);
     }
-
+    
```

Comment:
This isn't too vital, but trailing whitespace bugs me. Would you mind getting rid of this?

Reply:
thanks.  need to get clang-format working for me :-(

Reply:
Another option (which is, 99.9% of the time, what you want anyway) is just to add

```lisp
(add-hook 'before-save-hook (lambda () (delete-trailing-whitespace))
```

to your `.emacs.d/init.el` file.